egs2/chime7_task1/asr1/local/data.sh

```diff
@@ -26,7 +26,7 @@ background_snrs="20:10:15:5:0"
 gss_dsets=$(echo $gss_dsets | tr "," " ") # split by commas
 
 
-if [ $decode_only == 1 ]; then
+if [ -n "$decode_only" ]; then
```

Comment:
we can use ${skip_train} and ${test_sets} instead of decode_only.

Reply:
as before, we cannot change extensively the recipe at this point. I added the check for the argument. 
Note that this will also require to change all two README.md and notify participants.